src/installation/guides/chroot.md

```diff
@@ -0,0 +1,252 @@
+# mkdir -p "$VDROOT"/boot/efi
+# mount /dev/sda1 "$VDROOT"/boot/efi
+```
+
+Initiliazile swap space, if desired using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+[Select a mirror to download packages
+from.](../../xbps/repositories/mirror/index.md) For simplicity, save this URL to
+a shell variable for later use. Append `/current` to this repository url to
+install a glibc distribution, append `/current/musl` for a musl distribution (NB
+musl binaries are not available for i686).
+
+For example,
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+xbps also needs to know what architecture is being installed, available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers.
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Three metapackages are available to boot strap Void: `base-system`,
+`base-voidstrap` and `base-minimal`. `base-minimal` is a strict subset of
+`base-voidstrap` and `base-voidstrap` is a strict subset of `base-system`. You
+may wish to compare these packages for yourself. `xbps-query -x [package]` will
+list dependencies pulled in for the given package. Even `base-system` is a very
+minimal installation.
+
+`base-system` is the package that is installed when using the interactive
+installer. `base-voidstrap` is the package that is installed in rootfs tarballs.
+
+`base-minimal` and `base-voidstrap` do not come with a linux kernel. The `linux`
+package always has the newest available kernel. Alternatively, a long term
+support kernel can be used by installing `linux-lts`.
+
+NB `base-minimal` does not include a
+[terminfo(5)](https://man.voidlinux.org/terminfo.5) database, which will cause
+your terminal to behave in unexpected ways after entering the chroot. You may
+wish to install the package `ncurses-base` to get these files.
+
+If unsure, install `base-system`.
+
+```
+# PKGS="base-system"
+```
+
+Use xbps to boot strap the distribution
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r $VDROOT -R "$REPO" $PKGS
+```
+
+## Configuration
+
+### Edit Files
+
+Write the hostname to `$VDROOT/etc/hostname`. Go through options in
+`$VDROOT/etc/rc.conf`. If installing a glibc distribution, edit
+`$VDROOT/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+### Entering the Chroot
+
+Mount the pseudo-filesystems needed for a chroot,
+
+```
+# cd "$VDROOT"
+# mount -t proc /proc ./proc
+# mount -t sysfs /sys ./sys
+# mount -o bind /dev ./dev
+# mount -t devpts pts ./dev/pts
+```
+
+Copy the DNS configuration into the new root so that xbps can still download
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf ./etc
```

Comment:
an absolute path would be preferable

Reply:
okay, I've changed the preceding mount commands as well to have an absolute path, instead of cding into the new root directory. I used these relative paths so the commands are shorter, but I agree this is less prone to error for someone who's blindly copy-pasting commands (and those who know what they're doing can figure out that they can cd in there anyway if they're typing the whole thing out).